lib/UIKit/TUITextView.m

```
@@ -299,13 +300,6 @@ - (void)drawRect:(CGRect)rect
 		}];
 	}
 	
-	BOOL doMask = singleLine;
-	if(doMask) {
-		CGContextSaveGState(ctx);
-		CGFloat radius = floor(rect.size.height / 2);
-		CGContextClipToRoundRect(ctx, CGRectInset(textRect, 0.0f, -radius), radius);
```

Because adding a rounded rectangle mask to a text field doesn't "fit in" on Mac OS X. On iOS it'd blend in fine, but Cocoa, by default doesn't have masks. Perhaps it could be done using the `drawFrame` property if needed.